test/src/test/integration/librad/scenario/tracked_references.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+use crate::{
+    logging,
+    rad::{identities::TestProject, testnet},
+};
+use librad::{
+    git::{
+        identities,
+        storage::ReadOnlyStorage,
+        tracking,
+        types::{Namespace, Reference},
+        util::quick_commit,
+    },
+    git_ext::tree,
+    reflike,
+};
+use std::ops::Index as _;
+
+fn config() -> testnet::Config {
+    testnet::Config {
+        num_peers: nonzero!(2usize),
+        min_connected: 2,
+        bootstrap: testnet::Bootstrap::from_env(),
+    }
+}
+
+#[test]
```

Comment:
The attribute seems a bit lost with a big block of comment between it and the function it applies to.